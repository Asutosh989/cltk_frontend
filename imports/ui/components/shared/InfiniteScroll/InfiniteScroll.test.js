
import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

// component:
import InfiniteScroll from './InfiniteScroll';

describe('InfiniteScroll', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(
				<MuiThemeProvider>
					<InfiniteScroll />
				</MuiThemeProvider>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
