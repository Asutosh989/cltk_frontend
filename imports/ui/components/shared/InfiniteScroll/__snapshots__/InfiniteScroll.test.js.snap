// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfiniteScroll renders correctly 1`] = `
<span>
  <div>
     
  </div>
</span>
`;
